physics/basic_orbital_capture.py

```diff
@@ -0,0 +1,123 @@
+import math
+
+"""
+These two functions will return the radii of impact for a target object
+of mass M and radius R as well as it's effective cross sectional area σ(sigma).
+That is to say any projectile with velocity v passing within σ, will impact the
+target object with mass M. The derivation of which is given at the bottom
+of this file.
+
+The derivation shows that a projectile does not need to aim directly at the target
+body in order to hit it, as  R_capture>R_target. Astronomers refer to the effective
+cross section for capture as σ=π*R_capture**2.
+
+This algorithm does not account for an N-body problem.
+
+"""
+
+
+def capture_radii(
+    target_body_radius: float, target_body_mass: float, projectile_velocity: float
+) -> float:
+    # Gravitational constant to four significant figures as of 7/8/2023|
+    # Source google: gravitational constant
+    g = 6.6743e-11  # SI units (N*m**2)/kg**2
```

Comment:
Please use scipy's gravitational constant instead
https://docs.scipy.org/doc/scipy/reference/constants.html

Reply:
Sorry for being lazy and not reading doctest documentation. I've added the scipy constants, and actually made some doctests for the functions.